### DISCUSSION TITLE
English Translation

### DISCUSSION THREAD

**Discussion by carras** [Jeffery186/newifi3-d2-openwrt#6] [Q&A]
Hi there, I managed to translate the interface to English language.

But the Wireless section is in chinese, do you know how to make it in English?

Thanks.

**Comment by Jeffery186** [comment]
Sorry, the language package of the upstream default htm template is Chinese. It may be pure English that needs to be re-modified and then compiled, 
https://github.com/coolsnowwolf/lede/tree/049e623690fa27beca5f58fce7428605af9db8c2/package/lean/mt/luci-app-mtwifi/luasrc/view/admin_mtk
